egs/wsj/s5/steps/nnet3/decode_score_fusion.sh

```diff
@@ -110,6 +110,22 @@ if [ ! -z "$online_ivector_dir" ]; then
     ivector_opts="--online-ivectors=scp:$online_ivector_dir/ivector_online.scp --online-ivector-period=$ivector_period"
 fi
 
+# assign frame_subsampling_factor automatically if empty
+if [ -z $frame_subsampling_factor ]; then
+   frame_subsampling_factor=`cat ${model_dirs[0]}/frame_subsampling_factor` || exit 1;
+fi
+
+# check if standard chain system or not.
+if [ $frame_subsampling_factor -eq 3 ]; then
+   if [ $acwt != 1.0 ] || [ $post_decode_acwt != 10.0 ]; then
+     echo -e '\n\n'
+     echo "$0 WARNING: In stanard chain sysemt, acwt = 1.0, post_decode_acwt = 10.0"
```

Comment:
thanks.  Please check for typos, and confirm that you tested the script somehow.

Reply:
Thank you for advice.
The script did well in my tests.
1. Assign frame_subsampling_factor automatically.
2. Print WARNING when the setup is not the same as standard chain system.